runtime/lua/vim/secure.lua

```diff
@@ -1,5 +1,45 @@
 local M = {}
 
+--- Reads trust database from $XDG_STATE_HOME/nvim/trust.
+---
+--- @return (table) Contents of trust database, if it exists. Empty table otherwise.
+local function read_trust()
```

Comment:
Make this (and `write_trust`) private with `---@private`.

Reply:
Resolved in 379075cf4297986fc8ac03175eb2e7fc2eb1062f